src/Build/Microsoft.Build.csproj

```diff
@@ -28,6 +28,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\Framework\Microsoft.Build.Framework.csproj" />
+    <ProjectReference Include="..\Tasks\Microsoft.Build.Tasks.csproj" />
```

Comment:
I definitely agree, this was just a quick hack me and @cdmihai came up with for perf tests to make sure the serialization initializers are called early enough.

Reply:
Yeah. A lot of the "shared" code which is compiled into multiple assemblies is to avoid exactly this issue :( We'll need to look at a way to avoid this. And clearly we can't have this as "shared" code because it would then have to be initialized per assembly.